build_debian.sh

```diff
@@ -208,6 +208,23 @@ sudo LANG=C chroot $FILESYSTEM_ROOT apt-get update
 sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -y install docker-ce=${DOCKER_VERSION}
 sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -y remove software-properties-common gnupg2
 
+if [ "$INSTALL_KUBERNETES" == "y" ]
```

Comment:
>$INSTALL_KUBERNETES [](start = 6, length = 19)

You are expected to add INSTALL_KUBERNETES/KUBERNETES_VERSION samples into `rules/config`

Reply:
Good catch. Thanks. Updated.